gradle.properties

```diff
@@ -1,21 +1,16 @@
-# Project-wide Gradle settings.
-
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
 android.enableJetifier=true
 android.enableR8.fullMode=true
+android.injected.testOnly=false
 android.useAndroidX=true
+kapt.include.compile.classpath=false
 kapt.incremental.apt=true
-
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx1536m
+kapt.use.worker.api=true
+kapt.verbose=true
+kotlin.code.style=official
+kotlin.incremental=true
+kotlin.incremental.usePreciseJavaTracking=true
+kotlin.parallel.tasks.in.project=true
+org.gradle.caching=true
+org.gradle.configureondemand=true
+org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
+org.gradle.parallel=true
```

Comment:
What is this?

Reply:
Add gradle params to enhance speed of build，like parallel build, enlarge heap size, use cache ...